cmd/deploy/deploy.go

```diff
@@ -283,6 +309,19 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 	return nil
 }
 
+func setManifestEnvVars(service, registry, repository, digest string) {
+	os.Setenv(fmt.Sprintf("build.%s.registry", service), okteto.Context().Registry)
```

Comment:
Not sure if `${api.tag}` would be cleaner than `${build.api.tag}. @rlamana what do you think?